### ISSUE TITLE
vscode dropping debug.log in source directories (regression?)

### ISSUE THREAD

**Issue by HaraldKorneliussen** [microsoft/vscode#138245]
Issue Type: <b>Bug</b>

1. Open a terminal
2. Open a file from it 

    code example.txt

Expected result: file opens in new tab
Actual result: file opens in new tab, but the following is both printed to terminal, and added as a line in a new file called "debug.log" in the same folder the command was run in.

    [1201/094425.551:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)

This seems to be a regression of 
https://github.com/microsoft/vscode/issues/105191

VS Code version: Code 1.62.3 (ccbaa2d27e38e5afa3e5c21c1c7bef4657064247, 2021-11-17T08:11:14.551Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz (8 x 4200)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.93GB (11.19GB free)|
|Process Argv|--crash-reporter-id 9b668e8d-f3ed-4b70-8405-b49962d82164|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (24)</summary>

Extension|Author (truncated)|Version
---|---|---
alloy|Ara|0.7.0
pgformatter|bra|1.17.0
better-toml|bun|0.3.2
npm-intellisense|chr|1.4.0
vscode-eslint|dba|2.2.2
alloy-vscode|Don|0.1.6
vscode-npm-script|eg2|0.3.23
nuget-reverse-package-search|jes|0.1.68
sublime-babel-vscode|jos|0.2.10
dotenv|mik|1.0.1
vscode-docker|ms-|1.18.0
csharp|ms-|1.23.16
python|ms-|2021.11.1422169775
vscode-pylance|ms-|2021.11.2
jupyter|ms-|2021.10.1101450599
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.3
remote-containers|ms-|0.205.2
remote-wsl|ms-|0.58.5
powershell|ms-|2021.10.2
vscode-typescript-tslint-plugin|ms-|1.3.3
vscode-jscpd|pau|0.3.4
rust|rus|0.7.8
sort-lines|Tyr|1.9.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis300:30402085
vsaa593:30376534
vssld246:30401890
pythonvs932cf:30404740
vscexrecpromptc:30404946
vscop804:30404766
vscop453:30404998

```

</details>

**Comment by HaraldKorneliussen** [comment]
If there's more you need / anything I can do to help, I'm available.

**Comment by SURYASPR** [comment]
I'm seeing this issue too

**Comment by SamKingDev** [comment]
I'm now also seeing this issue.

**Comment by lgl017** [comment]
I happen this too.
https://github.com/microsoft/vscode-eslint/issues/1414#issue-1126759988

**Comment by deepak1556** [comment]
This should be addressed in latest stable `1.73`
